Resources/Prototypes/Entities/Objects/Specific/Hydroponics/nymph_seed.yml

```diff
@@ -0,0 +1,35 @@
+- type: entity
+  parent: SeedBase
+  name: pack of dionea spores
+  id: NymphSeed
+  components:
+    - type: Seed
+      seedId: NymphSeed
+    - type: Sprite
+      sprite: Objects/Specific/Hydroponics/nymph_seed.rsi
+
+
+- type: seed
+  id: NymphSeed
+  name: pack of dionea spores
+  noun: seeds-noun-seeds
+  displayName: pack of dionea spores
+  plantRsi: Objects/Specific/Hydroponics/nymph_seed.rsi
+  productPrototypes:
+    - MobNymph
+  lifespan: 50
+  maturation: 8
+  production: 3
```

Comment:
The maturation value should be higher and the production and yield values should be lower. As it stands, you can make an absurd amount of ghost roles using these.